### ISSUE TITLE
Typescript definition broken

### ISSUE THREAD

**Issue by upngo** [vinzscam/react-native-file-viewer#112]
index.d.ts does not have a default export and so attempting to call 
`import FileViewer from 'react-native-file-viewer';` shows
`TS1192: Module '"/Users/..../work/......./node_modules/react-native-file-viewer/index"' has no default export.`
 and it is unusable.

If you delete the index.d.ts and rely on index.js then it works (but obviously no types)

**Comment by vinzscam** [comment]
Yes, true. I will fix this.

It looks you are facing the issue because you have `allowSyntheticDefaultImports: false` in your tsconfig.
In the meantime these two ways of importing the package should also work for you:

```javascript
import * as FileViewer from 'react-native-file-viewer'
```

or

```javascript
import { open } from 'react-native-file-viewer'
```

**Comment by vinzscam** [comment]
Fixed by #123
